ckanext/cnra_schema/templates/scheming/package/snippets/package_form.html

```diff
@@ -1,9 +1,53 @@
 {% ckan_extends %}
 
+{% block basic_fields %}
+  {% block styles %}
+      <link rel="stylesheet" href="/css/additional_metadata_form.css" />
+  {% endblock%}
+
+  <p class=right-text>
+    Questions? See: <a href="/pages/metadata"><u>Metadta Guide</u></a>
+  </p>
+  
+  {%- if not dataset_type -%}
+    <p>
+    dataset_type not passed to template. your version of CKAN
+    might not be compatible with ckanext-scheming
+    </p>
+  {%- endif -%}
+  {%- set schema = h.scheming_get_dataset_schema(dataset_type) -%}
+  <!-- This for loop is intended to separate primary metadata and additiooanl metadata-->
+  {%- for field in schema.dataset_fields[:19] -%}
+    {%- if field.form_snippet is not none -%}
+      {%- snippet 'scheming/snippets/form_field.html',
+        field=field, data=data, errors=errors, licenses=c.licenses,
+        entity_type='dataset', object_type=dataset_type -%}
+    {%- endif -%}
+  {%- endfor -%}
+  <!-- HACK: A new accordion div is being used to hide non-primary metadatafields fields under Additional Fields accordion. ESN-1341-->
+  <div class="accordion-toggle" data-toggle="collapse" href="#collapse-Metadata">
+    <h2 class="accordion-heading">Additional Fields</h2>
+  </div>
+  <div id=collapse-Metadata class="collapse accordion-body">
+    {%- for field in schema.dataset_fields[19:] -%}
```

Comment:
Please leave a comment to state that the loop is to display additional metadata.